client/blocks/reader-full-post/_content.scss

```diff
@@ -286,6 +286,21 @@
 	.wp-block-embed__wrapper iframe[src*='icloud.com'] {
 		min-height: 300px;
 	}
+
+	.wp-block-latest-posts__list {
+		margin: 0px;
+		list-style: none;
+		margin-bottom: 4rem;
```

Comment:
Shall we try reducing the top and bottom margins a bit? I'm happy with it having similar spacing to a normal paragraph.

Reply:
Apologies, I wasn't quite ready for code review. Resolved now.

Reply:
Ok. Should be roughly the same now: 
![2020-08-06 16 03 34](https://user-images.githubusercontent.com/570876/89489502-c4b3bc80-d7fe-11ea-94b0-80fed8dff5b9.gif)